pkg/cmd/openshift-operators/webconsole-operator/webconsole_operator_trigger.go

```diff
@@ -0,0 +1,275 @@
+package webconsole_operator
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/blang/semver"
+	"github.com/golang/glog"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/informers"
+	appsclientv1 "k8s.io/client-go/kubernetes/typed/apps/v1"
+	coreclientv1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	apiregistrationclientv1beta1 "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1beta1"
+
+	operatorsv1alpha1 "github.com/openshift/origin/pkg/cmd/openshift-operators/apis/operators/v1alpha1"
+	"github.com/openshift/origin/pkg/cmd/openshift-operators/apis/operators/v1alpha1helpers"
+	webconsoleclientv1alpha1 "github.com/openshift/origin/pkg/cmd/openshift-operators/generated/clientset/versioned/typed/webconsole/v1alpha1"
+	webconsoleinformerv1alpha1 "github.com/openshift/origin/pkg/cmd/openshift-operators/generated/informers/externalversions/webconsole/v1alpha1"
+)
+
+const targetNamespaceName = "openshift-web-console"
```

Comment:
do we want to hardcode this? is ansible using the same namespace? (does it matter? ;-)

Reply:
> do we want to hardcode this? is ansible using the same namespace? (does it matter? ;-)

Yes we will hardcode them.  Yes ansible is using the same one.  No we don't care what ansible does since we'll migrate only once and can build the logic here.